### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+            return Response(serializer.data, status.HTTP_201_CREATED)
+
+        return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(["GET", "PUT", "DELETE"])
+def movie_detail(request, pk):
+    movie = get_object_or_404(Movie, pk=pk)
+
```
Comment (skrypka-dmytro) [inline cinema/views.py:31]: and here

File: cinema/views.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
```
Comment (skrypka-dmytro) [inline cinema/views.py:13]: add type annotation

**Review by skrypka-dmytro — CHANGES_REQUESTED** [review]

File: cinema/views.py (hunk 1 above)
Comment (HStakhiv) [reply cinema/views.py:31]: Disagree

File: cinema/views.py (hunk 2 above)
Comment (HStakhiv) [reply cinema/views.py:13]: Disagree

**Review by skrypka-dmytro — APPROVED** [review]

File: cinema/serializers.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField(required=False)
```
Comment (TPolina) [inline cinema/serializers.py:9]: if you want to make the description not a required field you need to specify this in models as well

**Review by TPolina — CHANGES_REQUESTED** [review]

File: cinema/serializers.py (hunk 3 above)
Comment (HStakhiv) [reply cinema/serializers.py]: I chose to make it required 😅

**Review by TPolina — APPROVED** [review]

**Pull request closed without merging**